test/test-dtutils.lua

```diff
@@ -0,0 +1,116 @@
+--[[
+
+    test-dtutils.lua - the start of a testing framework idea....
+
+    copyright (c) 2016 Bill Ferguson
+
+    TODO: Think this out a little better
```

Comment:
you might want to talk with @LebedevRI which is also thinking of doing something like that (but for DT, not specifically for lua)